pandas/core/arrays/datetimelike.py

```diff
@@ -1381,6 +1429,44 @@ def _ensure_localized(self, arg, ambiguous='raise', nonexistent='raise',
                 )
         return arg
 
+    # --------------------------------------------------------------
```

Comment:
This section should be in master now

Reply:
Thanks. FYI I'm doing a biggish cleanup right now. Should be done in an hour or so.